databaseFiles/tosReminderTable.js

```diff
@@ -4,18 +4,18 @@ const connect = require('./connect.js');
 const sequelize = connect.sequelize;
 
 module.exports = sequelize.define('TosReminder', {
-	user_id:{
+	user_id: {
         type: DataTypes.STRING,
         allowNull: false
     },
     joinTime: {
         type: DataTypes.DATE,
-        allowNull : false
```

Comment:
Could you fix the different indentations (tabs/spaces)?

Reply:
Fixed these with a bunch of other eslint fixes